src/components/ui/icon-button/icon-button.js

```diff
@@ -25,12 +25,29 @@ const IconButton = ({
 }) => {
   const Icon = icon;
 
+  const [isTooltipVisible, setIsTooltipVisible] = useState(false);
+
   const labelPosition = labelPositionTypes.includes(
     labelTextPosition.toLowerCase()
   )
     ? labelTextPosition.toLocaleLowerCase()
     : 'right';
 
+  const showTooltip = () => {
+    if (localStorage.getItem('delayShow') === null) {
+      window.localStorage.setItem('delayShow', true);
```

Comment:
hey i think we will need to have some naming convention for the storage here rather than just `delayShow`. Something like `kedro-viz-tooltip-show`, maybe best to discuss with the team?